translations/base-es.yaml

```diff
@@ -73,11 +73,11 @@ mainMenu:
     savegameLevel: Nivel <x>
     savegameLevelUnknown: Nivel desconocido
     savegameUnnamed: Sin nombre
-    puzzleMode: Puzzle Mode
-    back: Back
-    puzzleDlcText: Do you enjoy compacting and optimizing factories? Get the Puzzle
-        DLC now on Steam for even more fun!
-    puzzleDlcWishlist: Wishlist now!
+    puzzleMode: Modo Puzle
+    back: Atrás
+    puzzleDlcText: ¿Disfrutas compactando y optimizando fábricas?
+      ¡Consigue ahora el DLC de Puzles en Steam para aún más diversión!
```

Comment:
Seems like you removed two spaces in the front. Yaml is space sensitive, so please put them back, or it might throw up some errors